gcloud-java-contrib/gcloud-java-nio/pom.xml

```diff
@@ -77,24 +72,6 @@
       <artifactId>mockito-core</artifactId>
       <version>1.9.5</version>
     </dependency>
-    <dependency>
```

Comment:
I would like for this project to be its own artifact with a pom.xml file. I'm not sure what the value is in having aggregate jars. I hadn't heard of that until now.